install-dev/data/xml/hook_alias.xml

```diff
@@ -337,5 +337,14 @@
       <alias>actionAfterDeleteProductInCart</alias>
       <name>actionDeleteProductInCartAfter</name>
     </hook_alias>
+
+    <hook_alias id="adminOrderLeft">
```

Comment:
Hooks have moved on Edit an Order page so I gave them new name that suit better for their positioning. But I kept backward compatibility with hook aliases.